src/components/EndowmentScale.svelte

```diff
@@ -0,0 +1,86 @@
+<script>
+  // Component that draws a rating scale and synchronized pain duration bar to the
+  // screen
+  import { globalVars, calcPropSpent, round2, groupStore } from "../utils.js";
+
+  // INPUTS:
+  export let disabled = false;
+  export let questionText;
+  export let questionType = "other";
+  export let rating;
+  export let endowment;
+
+  $: whoText = questionType === "other" ? "They" : "You";
+  $: otherText = questionType === "other" ? "You" : "They";
+  $: endowmentAmount = endowment;
+
+  let multiplier = globalVars.multiplier;
+
+  let propSpent;
+  $: {
+    const { proportionOfEndowmentSpent } = calcPropSpent(rating, endowment);
+    propSpent = proportionOfEndowmentSpent;
+  }
+  // OUTPUTS:
+  // When using this component use bind:rating={someVariable} to get access to the
+  // rating value
+  // You can also optionally listen for on:change={someFunction} to run someFunction
+  // each time the user changes their rating
+</script>
+
+<div class="flex flex-col px-2 justify-items-center">
+  <p class="mb-2 text-lg">Shared endowment: ${endowment}</p>
+  <p class="mb-2 text-lg">Multiplier: {globalVars.multiplier}</p>
+  <h2 class="my-4 text-2xl">{questionText}</h2>
+</div>
+<!-- Money scale -->
+<div class="flex flex-col px-2 justify-items-center">
+  <label for="ratingScale" class="mb-2 text-xl"
+    >{whoText} would keep: ${round2(endowment - propSpent * endowment)}</label
```

Comment:
We can run this by Luke and will probs need to keep tweaking it, but I feel like there's a bit too much information on the screen right now and it would be great to simplify it. One thought is to remove the "you/they would keep" and "they/you would give" labels above and below the scale, and instead put that info in the anchors of the scale itself.

Reply:
ooooh good idea about the scale anchors. i forgot that could be an option! and yeaaaa i put it all there and then felt too attached to remove the summary and kept it for now but definitely needs pruning. hahaha

Reply:
579b771